sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBPublisher.java

```diff
@@ -66,30 +125,43 @@ private static void publish(
       builder.setDefaultCredentialsProvider(provider);
     }
 
-    final HttpPost postRequest = new HttpPost(settings.host + "/write?db=" + settings.database);
+    return builder;
+  }
 
-    final StringBuilder metricBuilder = new StringBuilder();
-    results.stream()
```

Comment:
nit: The original code with the strings looks uglier but somehow is easier to understand in a single read (so easier to maintain), the new one requires a lot of methods and jumping back and forth in code for not much. Can we go back to the older approach

Reply:
Hm...sure we can.